WORKFLOW-OSX-Machine-Setup-RoR.md

```diff
@@ -0,0 +1,87 @@
+# Basic OS X Machine Setup for RoR Development 
+####Pre-requisites
+1. Update to Yosemite(OSX 10.10)
+
+2. Install xCode 6.x & xCode Command Line Tools
```

Comment:
I was also wondering if it's part of pre-req steps or not. I still insist to suggest the sequence `xCode->xCode CLI->Homebrew` as they might encounter Homebrew complaining about http://xquartz.macosforge.org/landing/ not being there prior installation. Anyway, I'll detail these out plus add doing a `$ brew doctor` in the end to verify everything is in place before Ruby/Rails part of the setup.